src/components/networked.js

```diff
@@ -315,6 +316,21 @@ AFRAME.registerComponent('networked', {
 
   gatherComponentsData: function(fullSync) {
     var componentsData = null;
+    var euler_w = null;
+    var pos_w = null;
+    var quat_w = null;
+    var scale_w = null;
+
+    if (this.data.synchWorldTransforms === true) {
+      euler_w = new THREE.Euler();
+      pos_w = new THREE.Vector3();
+      quat_w = new THREE.Quaternion();
+      scale_w = new THREE.Vector3();
```

Comment:
You are creating those objects each time `gatherComponentsData` is called, not cool for the garbage collector. You may lose some fps when the garbage collector is triggered.